docs/users/index.md

```diff
@@ -6,13 +6,17 @@ For configuration options, please refer to our {doc}`operator's guide </operator
 
 ## Installation
 
-Jupyter Scheduler can be installed from the PyPI registry via `pip`:
+Jupyter Scheduler has a client extension and a server extension. **Both are required** to
+be able to schedule and run notebooks. If you install Jupyter Scheduler via the JupyterLab
+extension manager, you might only install the client extension and not the server extension.
+
+**Recommended:** Install Jupyter Scheduler from the PyPI registry via `pip`:
 
 ```
 pip install jupyter_scheduler
 ```
 
-This automatically enables its extensions. You can verify this by running
+This automatically enables the client and server extensions. You can verify this by running
```

Comment:
Nit: prefer lab extension over client extension to preserve parity with the rest of the Jupyter documentation. Though you could add a parenthetical to emphasize that lab extensions = client extensions.